File: CSharp/Unit1/03.ConsoleInputOutput/05.CompareWithoutIf/CompareWithoutIf.cs
/*
 * Write a program that gets two numbers from the console and 
 * prints the greater of them. Don’t use if statements.
*/
using System;
class CompareWithoutIf
{
    static void Main()
    {
        Console.Write("Input first digit : ");
        int firstDigit = int.Parse(Console.ReadLine());
        Console.Write("Input second digit : ");
        int secondDigit = int.Parse(Console.ReadLine());
        Console.WriteLine("The greater digit is " + 
            ((firstDigit > secondDigit) ? firstDigit : secondDigit ));
    }
}

File: Java/IntroJava/03.ConsoleInputOutput/src/CompareWithoutIf.java

import java.util.Scanner;
public class CompareWithoutIf {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		System.out.print("Input first digit : ");
        int firstDigit = input.nextInt();
        System.out.print("Input second digit : ");
        int secondDigit = input.nextInt();
        input.close();
        System.out.println("The greater digit is " + 
            ((firstDigit > secondDigit) ? firstDigit : secondDigit ));
	}
}
